integration/fuse/src/main/java/alluxio/jnifuse/struct/FuseFileInfo.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.jnifuse.struct;
+
+import java.nio.ByteBuffer;
+
+public class FuseFileInfo extends Struct {
```

Comment:
Don't all these public classes/methods require javadoc? Why isn't checkstyle complaining?

Reply:
i disabled style check for `jnifuse/`, as this intentionally preserves a lot C style to match linux kernel for readability.